lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -48,6 +48,11 @@
       - Command to execute when the container starts.
         A command may be either a string or a list.
         Prior to version 2.4, strings were split on commas.
+  nano_cpus:
+    description:
+        - Specify how much CPU resources a container can use.
```

Comment:
Although the name says "nano", it would be interesting to tell the unit. In this case is 1e-9.